scripts/dockerfiles/Dockerfile-base

```diff
@@ -34,6 +34,9 @@ RUN useradd -m --comment 'hcf user' vcap && \
 
 ADD monitrc.erb /opt/hcf/monitrc.erb
 
+ADD post-start.sh /opt/hcf/post-start.sh
+RUN chmod ug+x /opt/hcf/post-start.sh
```

Comment:
They can already run anything in the shell script as themselves (there's no suid involved), though? (This is mostly academic, though — it doesn't actually change anything right now, and we can flip the permissions when we need it.)